dockstore-webservice/src/main/java/io/dockstore/webservice/core/OrganizationUser.java

```diff
@@ -31,7 +30,6 @@ public class OrganizationUser implements Serializable {
 
     @ManyToOne
     @JoinColumn(name = "userId", insertable = false, updatable = false)
-    @JsonFilter(SLIM_USER_FILTER)
```

Comment:
As commented above, SLIM_USER_FILTER was referencing `@JsonIgnore` properties, so it was redundant.